modules/auxiliary/gather/piwigo_cve_2023_26876.rb

```diff
@@ -0,0 +1,168 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::SQLi
+  require 'metasploit/framework/hashes'
```

Comment:
Since you updated the `#check` method you can use this mixin to ensure that `run` is only called when the target is identified as being vulnerable.

```suggestion
  include Msf::Exploit::SQLi
  prepend Msf::Exploit::Remote::AutoCheck
  require 'metasploit/framework/hashes'
```